cypress/component/DataAccessRequest/data_access_request_validation.spec.js

```diff
@@ -0,0 +1,225 @@
+
+    cy.get('#btn_attemptSubmit').click();
+    cy.get('#btn_submit').click().then(() => {
+      expect(DAR.postDar).to.have.been.calledOnce;
+    });
+
+  });
+
+  it('Required fields should not be errored when you open page', () => {
+    cy.get('#piName').should('not.have.class', 'errored');
+    cy.get('#signingOfficial').should('not.have.class', 'errored');
+    cy.get('#itDirector').should('not.have.class', 'errored');
+    cy.get('#anvilUse').should('not.have.class', 'errored');
+
+    cy.get('#datasetIds').should('not.have.class', 'errored');
+
+    cy.get('#projectTitle').should('not.have.class', 'errored');
+    cy.get('#rus').should('not.have.class', 'errored');
+    cy.get('#nonTechRus').should('not.have.class', 'errored');
+
+    cy.get('#diseases').should('not.have.class', 'errored');
+
+    cy.get('#controls').should('not.have.class', 'errored');
+    cy.get('#population').should('not.have.class', 'errored');
+    cy.get('#oneGender').should('not.have.class', 'errored');
+    cy.get('#forProfit').should('not.have.class', 'errored');
+    cy.get('#pediatric').should('not.have.class', 'errored');
+    cy.get('#vulnerablePopulation').should('not.have.class', 'errored');
+    cy.get('#illegalBehavior').should('not.have.class', 'errored');
+    cy.get('#sexualDiseases').should('not.have.class', 'errored');
+    cy.get('#psychiatricTraits').should('not.have.class', 'errored');
+    cy.get('#notHealth').should('not.have.class', 'errored');
+    cy.get('#stigmatizedDiseases').should('not.have.class', 'errored');
+  });
+
+  it('Required fields get errors on submit', () => {
+    cy.get('#btn_attemptSubmit').click();
+
+    cy.get('#piName').should('have.class', 'errored');
+    cy.get('#signingOfficial').should('have.class', 'errored');
+    cy.get('#itDirector').should('have.class', 'errored');
+    cy.get('#anvilUse').should('have.class', 'errored');
+
+    cy.get('#datasetIds').should('have.class', 'errored');
+
+    cy.get('#projectTitle').should('have.class', 'errored');
+    cy.get('#rus').should('have.class', 'errored');
+    cy.get('#nonTechRus').should('have.class', 'errored');
+
+    cy.get('#diseases').should('have.class', 'errored');
+
+    cy.get('#controls').should('have.class', 'errored');
+    cy.get('#population').should('have.class', 'errored');
+    cy.get('#oneGender').should('have.class', 'errored');
+    cy.get('#forProfit').should('have.class', 'errored');
+    cy.get('#pediatric').should('have.class', 'errored');
+    cy.get('#vulnerablePopulation').should('have.class', 'errored');
+    cy.get('#illegalBehavior').should('have.class', 'errored');
+    cy.get('#sexualDiseases').should('have.class', 'errored');
+    cy.get('#psychiatricTraits').should('have.class', 'errored');
+    cy.get('#notHealth').should('have.class', 'errored');
+    cy.get('#stigmatizedDiseases').should('have.class', 'errored');
+  });
+
+  it('Internal / external / lab collaborators error properly', () => {
```

Comment:
one note: this test was wildly inconsistent (50/50 shot of passing), and I couldn't figure out why; it was failing upon calling a stubbed endpoint. then I changed something (still, am not sure what exactly) and it didn't fail after running it ~20 times. this is a potentially flaky test